packages/flutter_test/lib/src/widget_tester.dart

```diff
@@ -135,7 +141,10 @@ void testWidgets(
   TestVariant<Object?> variant = const DefaultTestVariant(),
   dynamic tags,
   int? retry,
+  LeakTesting? experimentalLeakTesting,
 }) {
+  // TODO(polina-c): enable leak tracking and update doc comment for `experimentalLeakTesting`.
+  // https://github.com/flutter/flutter/issues/135856
```

Comment:
Why not just implement this in this PR? Its odd to just add a parameter that doesn't do anything.

Reply:
I need to update leaking tests to set leak tracking off first, using this parameter.